testData/preferences/reset.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This is meant as a token for 'resetting' the system (which we needed for the review). The problem is that if user A logs in and then leaves the machine, there is no way to just "turn the system back to what it was". Logging this user in (or logging it in and out) will ensure that the system is back in it's original state.